Dockerfile

```diff
@@ -31,6 +31,18 @@ RUN mkdir -p /home/hive && \
     chgrp -R 0 /home/hive && \
     chmod -R g=u /home/hive
 
+# This is so that we can write source certificate anchors during container start up.
```

Comment:
Please add these to Dockerfile.dev as well.

Reply:
Done.